cmd/traefik/traefik.go

```diff
@@ -223,8 +211,23 @@ Complete documentation is available at https://traefik.io`,
 	os.Exit(0)
 }
 
-func run(globalConfiguration *configuration.GlobalConfiguration) {
-	fmtlog.SetFlags(fmtlog.Lshortfile | fmtlog.LstdFlags)
+func run(traefikConfiguration *TraefikConfiguration) {
```

Comment:
Is this something I should address in a follow-up PR?

Also, can you explain a bit the benefit you see in this? We were discussing with @Juliens that you want to keep `run` function to not know anything about the `traefikConfiguration`, because of a future feature you were working on. Is this still aligned with this effort?